.circleci/config.yml

```diff
@@ -127,7 +127,7 @@ jobs:
           name: Run percy snapshots
           command: |
             bundle exec jekyll build
-            npx percy snapshot _site/ --widths "1280" --ignore-files "assets/python/**"
+            npx percy snapshot _site/ --ignore-files "assets/python/**"
```

Comment:
Also need to ignore -> `parallel-universe/index.html` as this page is very long and this times out the percy build.